media/css/pebbles/components/google-search.scss

```diff
@@ -0,0 +1,113 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+
+// Custom Google search
+
+.moz-search {
+    @include clearfix;
+    background: $color-mozred;
+    color: #fff;
+    padding: 10px 0;
+    width: 100%;
+
+    .content {
+        @include clearfix;
+        margin: 0 auto;
+        padding: 0 10px;
+    }
+
+    .search-label {
+        @include font-size(16px);
+        display: block;
+        line-height: 1.25;
+        margin: 0 0 .5em;
+        padding: .25em 0;
+
+        @media #{$mq-tablet} {
+            float: left;
+            margin-bottom: 0;
+            width: 50%;
+        }
+    }
+
+    .search-label-term {
+        cursor: pointer;
+    }
+
+    .gsc-control-cse {
+        background: transparent !important;
+        border: 0 !important;
+        font-family: inherit !important;
+        padding: 0 !important;
+
+        table {
+            border-collapse: collapse;
+            margin: 0;
+        }
+
+        @media #{$mq-tablet} {
+            float: right;
+            width: 40%;
+        }
+    }
+
+    form.gsc-search-box {
+        margin: 0;
+        position: relative;
+    }
+
+    .gsc-search-box-tools .gsc-search-box .gsc-input,
+    .gsib_a {
+        padding: 0;
+    }
+
+    .gsc-input-box {
+        border-radius: 6px;
+        height: 2em;
+        padding: 2px 35px 2px 10px;
+
+        table {
+            height: 100%;
+        }
+    }
+
+    input#gsc-i-id1.gsc-input {
+        font-family: inherit !important;
+        min-height: 20px !important;
+        padding: .25em 0 !important;
+
+        &:focus {
+            box-shadow: none;
+        }
+    }
+
+    input.gsc-search-button {
+        border-radius: 5px;
+        margin: 0;
+        min-width: 0;
+        padding: 5px;
+        position: absolute;
+        right: 8px;
+        top: 4px;
+    }
+}
+
+.gsc-results-wrapper-overlay {
+    .gsc-table-result,
+    .gsc-thumbnail-inside,
+    .gsc-url-top {
+        padding: 0;
+    }
+
+    .gsc-result .gs-title {
+        height: auto !important;
+    }
+}
```

Comment:
The modal begins to get off center around 700px wide. Maybe add a media query?

![screen shot 2016-11-30 at 4 28 31 pm](https://cloud.githubusercontent.com/assets/317532/20774014/10a58236-b71a-11e6-9fbb-f879ef0fc89c.png)